doc/make.jl

```diff
@@ -43,6 +43,68 @@ cd(joinpath(@__DIR__, "src")) do
     end
 end
 
+# Because we have standard libraries that are hosted outside of the julia repo,
+# but their docs are included in the manual, we need to populate the remotes argument
+# of makedocs(), to make sure that Documenter knows how to resolve the directories
+# in stdlib/ to the correct remote Git repositories (for source and edit links).
+#
+# This function parses the *.version files in stdlib/, returning a dictionary with
+# all the key-value pairs from those files. *_GIT_URL and *_SHA1 fields are the ones
+# we will actually be interested in.
+function parse_stdlib_version_file(path)
```

Comment:
I blindly backported this part from PR #47105 by @mortenpi but it doesn't quite seem to work:
```
[ Info: HTMLWriter: rendering HTML pages.
ERROR: LoadError: MissingRemoteError: unable to generate source url
  path: /Users/mhorn/Projekte/Julia/julia.release-1.10/doc/src/stdlib/LibCURL.md
Documenter was unable to automatically determine the remote repository for this file.
This can happen if you are including docstrings or pages from secondary packages. Those packages
must be cloned as Git repositories (i.e. Pkg.develop instead Pkg.add), or the `remotes` keyword
must be configured appropriately. See the 'Remote repository links' section in the manual for
more information.
```

Reply:
Ah, I think this might be because [I can't build `release-1.10` on my mac right now](https://github.com/JuliaLang/julia/issues/60128) and thus was using the Julia 1.10.10 release to build the docs, but that breaks the code locating the stdlibs

I'll try it on another computer

Reply:
Yup that was it. Works fine now